src/components/footer/Footer.jsx

```diff
@@ -0,0 +1,39 @@
+import { BiGithub, BiTwitter, BiLinkedin } from "../../assets/icons/Icons";
+
+import "./footer.css";
+
+function Footer() {
+  return (
+    <footer className="footer d-flex align-center justify-center">
+      <p className="fs-s">Made with ❤️ by Sainath</p>
+      <div>
+        <a
+          href="/"
+          className="footer-link fs-l mr-sm"
+          target="_blank"
+          rel="noreferrer"
+        >
+          <BiGithub />
+        </a>
+        <a
+          href="/"
+          className="footer-link fs-l mr-sm"
+          target="_blank"
+          rel="noreferrer"
+        >
+          <BiTwitter />
+        </a>
+        <a
+          href="/"
+          target="_blank"
+          rel="noreferrer"
```

Comment:
Here you can create an array of objects and map that array because the structure is the same for all elements in the footer.

Reply:
Done.